### ISSUE TITLE
Can't run overlay locally

### ISSUE THREAD

**Issue by imagitama** [ThomasGaubert/zephyr#708]
I love the work you did to make Electron work with OpenVR!

**Describe the bug**

When I install everything locally and start Electron in dev I get exit code `4294930435`:

![image](https://user-images.githubusercontent.com/1967342/110566534-22402b00-818b-11eb-9d13-60cd7985cdf6.png)

**Comments**

I sprinkled some console.logs to help track it down and it appears to happen in/after `loadUrl` (I tried `loadFile` and same result):

![image](https://user-images.githubusercontent.com/1967342/110569125-08a0e280-818f-11eb-9978-af5911263d9e.png)

Also tried loading `https://github.com` and same error.

When I turn off overlay using `enableOverlay` config then it launches the Electron desktop windows fine so it's definitely the `VRWindow`.

**To Reproduce**

1. clone repo recursively
2. npm install
3. npm start

**Expected behavior**

The overlay would be rendered in SteamVR.

**Desktop**
 - OS: Windows 10 with Nodejs 12.16.3
 - Version: 2.0.0
 - Beta: yes - cloned latest version

**Android**

N/A

**Additional context**

Running in PowerShell as admin.

Installed packages:

```
electron@9.4.4
electron-builder@22.9.1
```

I did some digging and found [this SO question](https://stackoverflow.com/questions/65849599/electronjs-npm-err-errno-4294930435). I tried commenting out any mention of icons and it doesn't fix it.

**Comment by ThomasGaubert** [comment]
I've actually been dealing with the same thing recently 😃

I was able to get around the issue by upgrading Electron to 11.x and re-writing the node bindings with [node-addon-api](https://github.com/nodejs/node-addon-api). It's a huge (and painful) change, but I was able to run Zephyr and start communicating with SteamVR. There's still several bugs and more testing to be done before I push the code.
